.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
@smockle @dgreif @colebemis Would this change wreck anyone's vibe?

Reply:
Works for me!  I would have thought VS Code would use that version by default.  Were you running into issues without this?

Also, we had added some other settings in our `dropdown-with-zones` branch: https://github.com/primer/components/blob/d817a5317939420f9616219a3fe877ffe822b7aa/.vscode/settings.json.  Do we want to add those in as well?

Reply:
It never got ported over to the DropdownMenu branch, so I'd say add it here